pinot-integration-tests/pom.xml

```diff
@@ -36,6 +36,25 @@
   </properties>
 
   <profiles>
+    <profile>
```

Comment:
Do we need this section?

Reply:
I've adjusted it a bit, by adding the goal+phase, so it can become part of the build lifecycle. For the integration-test that's ok, as it is only 1 module.